_includes/navbar.html

```diff
@@ -13,12 +13,11 @@
     </div>
 
     <div class="collapse navbar-collapse" id="navbar-collapse">
-      <ul class="nav navbar-nav">
-        <li><a href="/all-posts"><i class="fa fa-list"></i> All Posts</a></li>
-        <!--<li><a href="/gallery"><i class="fa fa-photo"></i> Photo Gallery</a></li>-->
-      </ul>
       <ul class="nav navbar-nav navbar-right">
-        <!--<li><a href="#"> <i class="fa fa-star"></i> Join GDG Baroda</a></li>-->
+        <li><a href="#"><i class="fa fa-calendar"></i> Events</a></li>
```

Comment:
shall I design a functionality to show events separately or shall we keep it as a part of 'events' category in blog?